### PR TITLE
More stylcop warnings fixes on library project

### PR DESCRIPTION
**Pull request by valadas**
Mostly documentaiton and formatting. I recommend reviewer while hiding whitespace only changes :smile:

### CODE REVIEW

**Review by david-poindexter — APPROVED** [review]
Sweet - thanks @valadas 🎉

File: DNN Platform/Library/Common/Utilities/Config.cs [hunk 1]
```diff
@@ -1,925 +1,1044 @@
...
+            if (!string.IsNullOrEmpty(GetSetting("PersistentCookieTimeout")))
+            {
+                persistentCookieTimeout = int.Parse(GetSetting("PersistentCookieTimeout"));
+            }
+
+            return (persistentCookieTimeout == 0) ? GetAuthCookieTimeout() : persistentCookieTimeout;
+        }
+
+        /// <summary>
+        /// Gets a provider by it's type and name.
```
Comment (bdukes) [inline DNN Platform/Library/Common/Utilities/Config.cs:499]:
```suggestion
        /// Gets a provider by its type and name.
```

File: DNN Platform/Library/Common/Utilities/Config.cs [hunk 2]
```diff
@@ -1,925 +1,1044 @@
...
+        /// <returns>An emptry string upon success or the error message upon failure.</returns>
+        public static string Save(XmlDocument xmlDoc)
+        {
+            return Save(xmlDoc, "web.config");
+        }
+
+        /// <summary>
+        /// Save an XML document to the application root folder.
+        /// </summary>
+        /// <param name="xmlDoc">The configuraiton as an XML document.</param>
```
Comment (bdukes) [inline DNN Platform/Library/Common/Utilities/Config.cs:649]:
```suggestion
        /// <param name="xmlDoc">The configuration as an XML document.</param>
```

**Review by bdukes — APPROVED** [review]

**Comment by bdukes** [comment]
Good to merge once build is finished

File: DNN Platform/Library/Common/Utilities/Config.cs (hunk 1 above)
Comment (valadas) [reply DNN Platform/Library/Common/Utilities/Config.cs]: Damn english

**Pull request merged**